cell_feature_data/data_loader.py

```diff
@@ -326,8 +328,9 @@ def write_json_files(self, path: Path, write_megaset: bool = False) -> None:
         for file_name, data in json_files.items():
             file_path = path / file_name
             self.json_file_path_dict[file_name] = file_path
-            with open(file_path, "w") as f:
-                json.dump(data, f, indent=4)
+            if not file_path.exists():
```

Comment:
if file path exists then do nothing? is it worth printing a warning or no?

Reply:
The check is actually used when creating megasets. In scenarios where a user may return to a previously created folder to make edits or additions to the top-level dataset.json, we want to ensure the file(particularly the `datasets` array) won't be overwritten. 
For better readability, I can add a log message if the `file_path` already exists. Thank you for pointing it out!   


(Also, this made me wonder if we should support users editing the dataset folders by running the script at all. It seems going directly into the created files and manually modifying the values is much quicker. We can easily make this change if folks lean towards the latter approach)